karma.conf.js

```diff
@@ -21,7 +21,7 @@ module.exports = function(config) {
     basePath: "",
     frameworks: ["mocha"],
     files: [
-      "./public/js/**/tests/*"
+      "./public/js/**/tests/*.js"
```

Comment:
I opted to move `utils` up to `test-utils` as changing all of the test files to `*.spec.js` seemed like over kill at this point. We can re-evaluate at a later point. Also, there is nothing intrinsically wrong with naming the file `test-utils.js`.